notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/notificationhubs/NotificationHub.java

```diff
@@ -68,7 +66,7 @@ public static void initialize(Context context, String hubName, String connection
      *                needs access to.
      * @param manager A client that can create/overwrite a reference to this device with a backend.
      */
-    public static void initialize(Context context, InstallationManager manager) {
+    public static void initialize(Context context, InstallationAdapter manager) {

```

Comment:
If we're going to change the parameter type, might as well change the name.  Make sure to change all instances in all overloads

Reply:
Fixed in 04d821d1d383f723ec10deedb5ae0207872b54fb